manifests/ha/base/redis-ha/chart/values.yaml

```diff
@@ -5,8 +5,11 @@ redis-ha:
     masterGroupName: argocd
     config:
       save: "\"\""
+      bind: "0.0.0.0"
   haproxy:
     enabled: true
+    IPv6:
```

Comment:
We should document that IPV6 only clusters are not supported by Redis-HA.

Reply:
can definitely do that, any suggestions on the location?

Reply:
Just as an FYI, this has been implicitly true since at least v2.0.0. You can see the issue / PR I created last time [here](https://github.com/argoproj/argo-cd/issues/5957).

Reply:
You should be able to mention about IPV6 only clusters not being supported here `https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/high_availability.md`

Reply:
done!